lib/kafo/data_type_parser.rb

```diff
@@ -15,7 +15,7 @@ def initialize(manifest)
         line = line.force_encoding("UTF-8").strip
         next if line.start_with?('#')
```

Comment:
I'd prefer to implement this
```suggestion
        next if line.start_with?('#') || line.empty?
```